doc/main/reference/blocking_terminate.rst

```diff
@@ -60,16 +62,42 @@ Synopsis
 Member Functions
 ----------------
 
+.. cpp:function:: task_scheduler_handle()
+
+    **Effects**: Creates an instance of the ``task_scheduler_handle`` class that does not contain any reference to the task scheduler.
+
+-------------------------------------------------------
+
 .. cpp:function:: ~task_scheduler_handle()
 
     **Effects**: Destroys an instance of the ``task_scheduler_handle`` class.
     Releases a reference to the task scheduler and deactivates an instance of the ``task_scheduler_handle`` class.
 
 -------------------------------------------------------
 
+.. cpp:function:: task_scheduler_handle(task_scheduler_handle&& other) noexcept
+
+    **Effects**: Creates an instance of the ``task_scheduler_handle`` class that references the task scheduler referenced by other. In turn, ``other`` releases its reference to the task scheduler.
+
+-------------------------------------------------------
+
+.. cpp:function:: task_scheduler_handle& operator=(task_scheduler_handle&& other) noexcept
+
+    **Effects**: Releases a reference to the task scheduler referenced with ``this``. Adds a reference to the task scheduler referenced with ``other``.
```

Comment:
I think, we can rewrite it as we did above:

 **Effects**: Releases a reference to the task scheduler referenced by ``this``. Adds a reference to the task scheduler referenced by ``other``.

Reply:
```suggestion
    **Effects**: Releases a reference to the task scheduler referenced by ``this``. Adds a reference to the task scheduler referenced by ``other``.
```